scripts/kvm-test.py

```diff
@@ -451,9 +464,8 @@ def install(ctx):
         # appends += ['subiquity-channel=edge']
 
         with mounter(iso, mntdir):
-            appends += grub_get_extra_args(mntdir)
-
             if len(appends) > 0:
+                appends += grub_get_extra_args(mntdir)
```

Comment:
If we have modified the iso then I think we still need to go looking for the new layer, right?  So we would want this all the time, even if there aren't any other reasons to add to the kernel `-append` list?

Reply:
livefs-edit edits the grub config to point at the new layer (it should probably edit the initrd but well)